setup.py

```diff
@@ -4,15 +4,16 @@
 
 setup(
     name='api-integration',
-    version='1.4.1',
+    version='1.5.0',
```

Comment:
@jcdyer  Sorry I didn't notice this earlier -- this is a pretty minor change, so it makes more sense as a minor version update, say `1.4.2`?

Reply:
I've generally seen it done where, after 1.0, the second digit should be the minor release: Major.Minor.Patch, so bugfixes go under Patch, backwards compatible feature changes under minor, and backwards incompatible changes under Major.  But I'm happy to make a 1.4.2 release.